site/content/en/docs/concepts/admission_check.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Admission Check"
+date: 2023-10-05
+weight: 6
+description: >
+  A mechanism allowing custom internal or external components to control the timing of
+  workloads admission.
+---
+
+## Components
+
+### Admission Check
+
+Admission check is a none namespaced API object used to define details about an `AdmissionCheck` like:
+
+#### controllerName
+
+Is the name with which controller identifies with, not necessarily a K8S Pod or Deployment name. Cannot be empty.
+
+#### retryDelayMinutes
+
+Specifies how long to keep the workload suspended after a failed check (after it transitioned to False). After that the check state goes to "Unknown". The default is 15 min.
+
+#### parameters
+
+Identifies the resource providing additional check parameters.
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: AdmissionCheck
+metadata:
+  name: prov-test
+spec:
+  controllerName: kueue.x-k8s.io/provisioning-request
+  retryDelayMinutes: 15
+  parameters:
+    apiGroup: kueue.x-k8s.io
+    kind: ProvisioningRequestConfig
+    name: prov-test-config
+```
+
+### ClusterQueue admissionChecks
+
+Once defined an AdmissionCheck can be referenced in the ClsterQueues Spec resulting in all Workloads associated with the queue needing to be evaluated by its controller before being admitted. Similarly with `ResourceFalvors`, if an `AdmissionCheck` is not found or its controller has not mark it as `Active`, the cluster queue will be marked as Inactive.
+
+### AdmissionCheckState
+
+AdmissionCheckState is the way the way the state of an AdmissionCkeck for a specific Workload is tracked.
+
+It is following the schema:
+```yaml
+properties:
```

Comment:
Let's have an example instead.

"The status of a Workload that has pending AdmissionChecks looks like the following:"

Reply:
done

Reply:
done